src/views/blocks/tickets/extra-available.php

```diff
@@ -66,7 +63,7 @@
 	class="tribe-common-b3 tribe-tickets__item__extra__available"
 >
 	<?php if ( $show_unlimited && -1 === $available ) : ?>
-		<?php $this->template( 'blocks/tickets/extra-available-unlimited', array( 'ticket' => $ticket, 'key' => $key ) ); ?>
+		<?php $this->template( 'blocks/tickets/extra-available-unlimited', [ 'ticket' => $ticket, 'key' => $key ] ); ?>
```

Comment:
* :no_entry_sign: Short array syntax is not allowed
* :no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 08d9198da7360f39fd5ebf7db23d02e1</sub>

Reply:
@sc0ttkclark how do we fix this to allow short arrays?